powerauth-push-model/src/main/java/io/getlime/push/model/request/CreateDeviceRequest.java

```diff
@@ -16,81 +16,43 @@
 package io.getlime.push.model.request;
 
 import io.getlime.push.model.enumeration.MobilePlatform;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
 
 /**
  * Request object used for device registration.
  *
  * @author Petr Dvorak, petr@wultra.com
  */
+@Getter
+@Setter
 public class CreateDeviceRequest {
 
-    private String appId;
-    private String token;
-    private MobilePlatform platform;
-    private String activationId;
-
-    /**
-     * Get app ID associated with given device registration.
-     * @return App ID.
-     */
-    public String getAppId() {
-        return appId;
-    }
-
-    /**
-     * Set app ID associated with given device registration.
-     * @param appId App ID.
-     */
-    public void setAppId(String appId) {
-        this.appId = appId;
-    }
-
     /**
-     * Get APNs / FCM push token.
-     * @return Push token value.
+     * Application ID.
      */
-    public String getToken() {
-        return token;
-    }
-
-    /**
-     * Set APNs / FCM push token.
-     * @param token Push token value.
-     */
-    public void setToken(String token) {
-        this.token = token;
-    }
-
-    /**
-     * Get the platform name, either "ios" or "android".
-     * @return Platform name, "ios" or "android".
-     */
-    public MobilePlatform getPlatform() {
-        return platform;
-    }
+    @NotBlank
+    @Schema(description = "Application ID.")
+    private String appId;
 
     /**
-     * Set the platform name.
-     * @param platform Platform name.
+     * The push token is the value received from APNS, or FCM services without any modification.
      */
-    public void setPlatform(MobilePlatform platform) {
-        this.platform = platform;
-    }
+    @NotBlank
+    @Schema(description = "The push token is the value received from APNS, or FCM services without any modification.")
+    private String token;
 
-    /**
-     * Get PowerAuth 2.0 activation ID associated with given device registration.
-     * @return Activation ID.
-     */
-    public String getActivationId() {
-        return activationId;
-    }
+    @NotNull
+    private MobilePlatform platform;
 
     /**
-     * Set PowerAuth 2.0 activation ID associated with given device registration.
-     * @param activationId Activation ID.
+     * Activation ID.
      */
-    public void setActivationId(String activationId) {
-        this.activationId = activationId;
-    }
+    @NotBlank
+    @Schema(description = "Activation ID.")
+    private String activationId;
```

Comment:
We can use a more precise validation, because `activationId` format is well defined.